api/src/test/java/org/pmiops/workbench/rdr/RdrExportServiceImplTest.java

```diff
@@ -318,6 +319,16 @@ public void exportWorkspace() throws ApiException {
     verify(mockRdrApi).exportWorkspaces(ImmutableList.of(rdrWorkspace), NO_BACKFILL);
   }
 
+  @Test
```

Comment:
this test passes even without the code change.  What is this intended to demonstrate?

Reply:
ah, that hit `if (!rdrWorkspacesList.isEmpty()) ` block because I only have 1 workspace and it fails. 
Changed to 2 workspace, with one fail. Not the test is corrected that only one workspace is exported. (not having my change it returns 2).